src/commands/createFunctionApp/createCreateFunctionAppComponents.ts

```diff
@@ -51,7 +51,13 @@ export async function createCreateFunctionAppComponents(context: ICreateFunction
 
     await detectDockerfile(context);
 
-    promptSteps.push(new SiteNameStep(context.dockerfilePath ? "containerizedFunctionApp" : "functionApp"));
+    // SiteNameStep pre-requisites
```

Comment:
Please double check this location logic to make sure these are the scenarios where each type of list step applies